rules/consistent-indexof-check.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+const evaluateLiteralUnaryExpression = require('./utils/evaluate-literal-unaryexpression.js');
+const resolveVariableName = require('./utils/resolve-variable-name.js');
+
+const MESSAGE_ID = 'consistent-indexof-check';
+const messages = {
+	[MESSAGE_ID]: 'Prefer `{{replacement}}` over `{{value}}`.',
+};
+
+const comparisonMap = {
+	'<': '>',
+	'<=': '>=',
+	'>': '<',
+	'>=': '<=',
+};
+
+/**
+Check if the node is a call expression of `indexOf` or `lastIndexOf` method.
+
+@param {import('estree').Node} node
+@returns {node is import('estree').CallExpression}
+*/
+function isIndexOfCallExpression(node) {
```

Comment:
Please switch to use `isMethodCall`.

Reply:
b24f0c24f74d114dce607572e504653d334515a7